core/quantity/README.md

```diff
@@ -1,11 +1,232 @@
 # @itwin/core-quantity
 
-Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md for license terms and full copyright notice.
+- [@itwin/core-quantity](#itwincore-quantity)
+  - [Terms and Concepts](#terms-and-concepts)
+    - [Common Terms](#common-terms)
+    - [Concepts](#concepts)
+      - [Units Provider](#units-provider)
+      - [Unit Conversion](#unit-conversion)
+    - [Examples of Usage](#examples-of-usage)
+      - [Numeric Format](#numeric-format)
+      - [Composite Format](#composite-format)
+      - [Parsing Values](#parsing-values)
+      - [Mathematical Operation Parsing](#mathematical-operation-parsing)
+  - [Contributing](#contributing)
+- [Licensing](#licensing)
+
+The __@itwin/core-quantity__ package contains classes for quantity formatting and parsing. For detailed API documentation, see our [iTwin.js references documentation](https://www.itwinjs.org/reference/core-quantity/quantity/).
+
+Also check the [iTwin.js learnings documentation](https://www.itwinjs.org/learning/frontend/quantityformatting/#quantity-package) explaining quantity formatting and it's basic concepts.
+
+## Terms and Concepts
+
+### Common Terms
+
+- `Unit`/`UnitProps` - A named unit of measure which can be located by its name or label. The definition of any unit is represented through it's `UnitProps`.
+- `UnitsProvider` - An interface that locates the UnitProps for a unit given name or label. This interface also provides methods for `UnitConversion` to allow converting from one unit to another.
+- Unit Family/`Phenomenon` - A physical quantity that can be measured (e.g., length, temperature, pressure).  Only units in the same phenomenon can be converted between.
+- Persistence Unit - The unit used to store the quantity value in memory or to persist the value in an editable IModel.
+- Format/FormatProp - The display format for the quantity value. For example, an angle may be persisted in radians but formatted and shown to user in degrees.
+  - CompositeValue - An addition to the format specification that allows the explicit specification of a unit label, it also allows the persisted value to be displayed as up to 4 sub-units. Typical multi-unit composites are used to display `feet'-inches"` and `degree°minutes'seconds"`.
+- `FormatterSpec` - Holds the format specification as well as the `UnitConversion` between the persistence unit and all units defined in the format. This is done to avoid any async calls by the UnitsProvider during the formatting process.
```

Comment:
```suggestion
- `FormatterSpec` - Holds the format specification as well as the `UnitConversion` between the persistence unit and all units defined in the format. This is done to avoid any async calls by the `UnitsProvider` during the formatting process.
```

Reply:
addressed